packages/twenty-server/src/engine/workspace-manager/workspace-sync-metadata/constants/standard-field-ids.ts

```diff
@@ -155,6 +155,8 @@ export const CONNECTED_ACCOUNT_STANDARD_FIELD_IDS = {
   calendarChannels: '20202020-af4a-47bb-99ec-51911c1d3977',
   handleAliases: '20202020-8a3d-46be-814f-6228af16c47b',
   scopes: '20202020-8a3d-46be-814f-6228af16c47c',
+  customConnectionParams: '20202020-a1b2-46be-814f-6228af16c481',
```

Comment:
- Debating whether this should live on connectedAccount or messagingChannel (I don't have a strong opinion).
- If this lives on connectedAccount we'll probably need to store IMAP + CalDav + SMTP in that same field so we need to anticipate this in the json structure to avoid a future data migration ; and even if this lives in messaging channel we would probably still have SMTP+IMAP so wrapping it in properties would still make sense to make it future-proof
- `custom` carries a strong meaning in our codebase (means non-standard, like standard vs custom object) so I would get rid of it. Also we try to use full name, we've used Params much in the code, but IMO it's a mistake and we should have let it pass, it should be connectionSettings or connectionParameters

Reply:
OK for 1, will ask the team for their opinion and get back to you. Comment 2/3 still valid

Reply:
Quick reminder to address 2. IMO we should even rename some of the places that refer to this account as IMAP to IMAP_SMTP_CALDAV to make it future proof when we're talking about the general account and not too narrowly focused on IMAP. Same for the frontend endpoint, if the endpoint is too specific to IMAP that could be an issue and require additional refactoring later on.

Thanks a lot!